### PR TITLE
[57_2] Remove ispell from libmogan

### PR DESCRIPTION
**Pull request by Charonxin**
## Why you open this Pull Request?

Just delete Ispell plugin or move to another dir?

## What work have you done in the current Pull Request?

- [x] comment ispell related code

### CODE REVIEW

File: xmake.lua [hunk 1]
```diff
@@ -500,7 +500,7 @@ target("libmogan") do
     add_files(plugin_freetype_srcs)
     add_files(plugin_database_srcs)
     add_files(plugin_ghostscript_srcs)
-    add_files(plugin_ispell_srcs)
+    -- add_files(plugin_ispell_srcs)
```
Comment (jingkaimori) [inline xmake.lua:503]: better method is setting `plugin_ispell_srcs` to an empty list

**Review by jingkaimori — CHANGES_REQUESTED** [review]

File: xmake.lua (hunk 1 above)
Comment (darcy-shen) [reply xmake.lua:503]: > better method is setting plugin_ispell_srcs to an empty list

No. mutable variable is not a good practice

Comment (jingkaimori) [reply xmake.lua:503]: > mutable variable is not a good practice

My suggestion is to change definition of `plugin_ispell_srcs`:
```lua
    local plugin_ispell_srcs = {}
    -- .....
    add_files(plugin_ispell_srcs)
```

Comment (darcy-shen) [reply xmake.lua:503]: 这样也不合理，明明是 plugin_ispell_srcs，但是里面没有源代码文件

**Pull request closed without merging**